packages/dapp/src/i18n/en.json

```diff
@@ -48,5 +48,17 @@
     "info": "You’re using Testnet",
     "testnet": "Any transactions or actions performed on the Testnet will not be reflected on the Mainnet.",
     "switchToMainnet": "Switch to Mainnet"
+  },
+  "multisig": {
+    "connected": {
+      "title": "Multisig wallet connected",
+      "description": "Please notify other wallet participants to be ready to sign.",
+      "cta": "Okay"
+    },
+    "transactionInitiated": {
+      "title": "Multiple signatures required",
+      "description": "Please notify other multisig wallet participants to sign before the transaction expires.",
+      "cta": "Okay"
```

Comment:
Let's add a top-level key `button` and put this `Okay` there to avoid duplicated translations.